.travis.yml

```diff
@@ -1,4 +1,8 @@
+dist: xenial
 language: java
+jdk:
+  - oraclejdk11
```

Comment:
`openjdk11` should be the preferred `jdk` environment to use for Java 11 (due to the high-volatility of `oraclejdk` builds) - see https://github.com/openmicroscopy/bioformats/pull/3286#discussion_r240145628